### DISCUSSION TITLE
Error while building from source in termux

### DISCUSSION THREAD

**Discussion by jakbin** [inspircd/inspircd#1985] [Support]
I got this error .

![Screenshot_20220604-235624_Termux.jpg](https://user-images.githubusercontent.com/65591973/172032597-227ea46c-17ae-4de2-8e1b-de2933a1d0fc.jpg)

**Comment by ploxxusVulgaris** [comment]
try:

(1) sed -i 's/getdtablesize\(\)/sysconf\(_SC_OPEN_MAX\)/' src/coremods/core_oper/cmd_restart.cpp

(before make)

(2) compiles without errors

(3) profit

**Reply by jakbin** [reply]
this issue has been solved.
